database/dummyData/userEventData.js

```diff
@@ -214,97 +214,170 @@ const simulateUserEvents = (numOfSessions) => {
-            session: event.session,
-            events: [event.query],
-          },
-        });
-        generateEvents(num + 1);
-      })
-      // If the event just added to DB has id == 4, grab all assiociated session data
-      .then(() => {
-        if (event.query.eventId === 4) {
-          request.get(`http://localhost:3000/${event.session.id}`, (err, res, body) => {
-            if (err) throw err;
-            const parsedBody = JSON.parse(body);
-            const msg1 = {
-              userId: parsedBody.userId,
-              groupId: parsedBody.groupId,
-              events: parsedBody.events.reduce((prev, curr) => {
-                return curr.eventId === 3 ?
-                  prev.concat({
-                    movie: {
-                      id: curr.movieObj.id,
-                      profile: curr.movieObj.profile,
-                    },
-                    progress: curr.progress,
-                    startTime: new Date(),
-                  })
-                  : prev;
-              }, []),
-            };
-            // console.log('Message 1 for bus:', msg1);
-            const msg2 = {
-              userId: parsedBody.userId,
-              groupId: parsedBody.groupId,
-              recs: parsedBody.events.reduce((prev, curr) => {
-                return curr.eventId === 3 && curr.movieObj.isRec === true && curr.progress === 1 ?
-                  prev + curr.progress : prev;
-              }, 0),
-              nonRecs: parsedBody.events.reduce((prev, curr) => {
-                return curr.eventId === 3 && curr.movieObj.isRec === false && curr.progress === 1 ?
-                  prev + curr.progress : prev;
-              }, 0),
-            };
-            // console.log('Message 2 for bus:', msg2);
-            const msg3 = {
-              userId: parsedBody.userId,
-              // { movie: {id}, progress, startTime/endTime? }
-              events: parsedBody.events.reduce((prev, curr) => {
-                return curr.eventId === 3 ?
-                  prev.concat({
-                    movie: { id: curr.movieObj.id },
-                    progress: curr.progress,
-                    startTime: new Date(),
-                  })
-                  : prev;
-              }, []),
-            };
-            // console.log('Message 3 for bus:', msg3);
-          });
-        }
-      })
-      .catch(() => {
-        console.log('Error generating events');
+let index = -1;
+cron.schedule('*/1 * * * * *', () => {
+  index += 1;
+  if (index < eventArray.length) {
+    axios.post('http://localhost:3000/newEvent', eventArray[index])
+      .catch((err) => {
+        console.log('axios post error:', err);
       });
   } else {
-    dashboard.elasticCreate(eventDashboard);
-    eventArray = simulateUserEvents(100);
-    eventDashboard = [];
+    eventArray = simulateUserEvents(1);
+    index = -1;
   }
-};
+});
+
+// **** FOR SEED DATA GENERATION *****
+
+// const elasticInsert = () => {
```

Comment:
Commenting out this part makes the code look somewhat unmeaningful/purposeless. Maybe you could just comment out the part that actually runs this code (i.e. the cron worker)?